src/libraries/Common/src/Interop/Windows/Kernel32/Interop.DeviceIoControl.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Kernel32
+    {
+        // https://docs.microsoft.com/windows/win32/api/winioctl/ni-winioctl-fsctl_get_reparse_point
+        internal const int FSCTL_GET_REPARSE_POINT = 0x000900a8;
```

Comment:
Since this const is only used by the P/Invoke added in this same file, it makes sense to keep it here. We can move it in the future to its own file if the const is used by more than one P/Invoke.